### DISCUSSION TITLE
NexoPOS 5.2.6

### DISCUSSION THREAD

**Discussion by nexopos** [Blair2004/NexoPOS#2031] [Releases]
We're publishing this update as one of the last update of the branch version v5.2.x. The upcoming version v5.3.x will ship an major accounting upgrade. We've revamped the UI and the accounting principle to ensure it gives accurate figures, allowing owner to have a better understanding of their business.

Meanwhile, this update comes with few fixes that makes premium modules compatible with the v5.x branch. We're targetting here:

- [Stock Transfers](https://my.nexopos.com/en/marketplace/item/stock-transfers-module)
- [Multistore](https://my.nexopos.com/index.php/en/marketplace/item/nexo-multistore-module)
- [Racks Manager](https://my.nexopos.com/index.php/en/marketplace/item/racks-manager)

Changelog:
=======
- Added: new hook action
- Updated: better fields support on order settings
- Fixed: old uncleaned invoice template
- Added: new js action hook "ns-after-load-order"
- Added: permission is not needed to convert product
- Fixed: accuracy on math calculation
- Fixed: backspace has a correct behavior on the payment popup
- Fixed: first slice layaway amount.
- Updated: added a proper way to restrict order deletion on CRUD
- Updated: while loading taxes, we've turned the default "id" to "tax_id" as "id" is reserved for order taxes id.
- WIP: tax history should record every transaction including new payment
- Added: Hook filter to allow to customize payment types that trigger a register history.
- Fixed: Ensure while reverb is disabled it doesn't trigger any JS error
- Fixed: discount issue on cart applied to each product
- WIP: test case for register history
- Removed: deprecated snippet
- Fix https://github.com/Blair2004/NexoPOS/issues/1976: Incorrect URL for creating coupons
- Added: Support for props on AsyncComponent loaded on Popup (vue3)
- Removed: redundant options for Tax group selection
- Refactoring: RegisterHistory constants
- Added: new CashRegisterHIstoryBeforeDeletedEvent
- Refactoring: CashRegisterCashingFields and CashRegisterCashoutFields
- Added: support for account id on cashing and cashout, methods of CashRegisterService
- Deprecating: some classes and methods
- WIP: how cash register history is created/updated
- Added: new relation (order) to RegisterHistory
- Added: new account on demo
- Fixed: nexopos_payments_types to be truncated during the reset action
- Added: new accounting settings
- Fixed: route binding for accounting accuont modification
- Added: selecting transaction account on cash register action

We're also thankful for the recent contributions we got from:

- @nirzaf 
- @bilton30
- And others.

<hr /><em>This discussion was created from the release <a href='https://github.com/Blair2004/NexoPOS/releases/tag/v5.2.6'>NexoPOS 5.2.6</a>.</em>

**Comment by MONKE121** [comment]
Hi , is this free to download? and how to install it please I need a POS system on my computer I am not good installing this stuff.

**Reply by nzian** [reply]
Yes its free open source pos. Its core version open source and can clone the repository and and follow the instruction to install it. But it has some paid modules as well.
Thank you

**Reply by MONKE121** [reply]
Please teach me how to install it I need one

**Reply by nzian** [reply]
https://my.nexopos.com/en/documentation/getting-started/0-requirements
you can go thorugh the webiste.
